src/common/components/cards/instance-details.tsx

```diff
@@ -65,8 +65,17 @@ export const InstanceDetails = NamedFC<InstanceDetailsProps>('InstanceDetails',
         selected: result.isSelected,
     });
 
+    const kebabMenuAriaLabel = `More Actions ${result.identifiers.identifier} ${result.ruleId}`;
     return (
-        <div className={instanceDetailsCardStyling} tabIndex={0} onClick={cardClickHandler} onKeyDown={cardKeyPressHandler}>
+        <div
+            className={instanceDetailsCardStyling}
+            tabIndex={0}
+            onClick={cardClickHandler}
+            onKeyDown={cardKeyPressHandler}
+            aria-selected={result.isSelected}
+            aria-label={`${result.identifiers.identifier} card`}
+            role="row"
```

Comment:
Adding this comment base on @DaveTryon comment on teams:
Why is `role="row"' necessary here?

Reply:
Because `aria-selected` only works with certain `roles` and `row` seems to be the most appropriate for the card here.
https://www.w3.org/WAI/PF/aria/states_and_properties#aria-selected

Reply:
Using [role="row"](https://www.w3.org/WAI/PF/aria/roles#row) requires being contained in a rowgroup, grid, or treegrid... the current state results in `A11YSelfValidator.validate();` automated checks failures when rows are visible. 

We probably want an E2E test that does a check on the cards UX with some rules expanded.